BUILD.bazel

```diff
@@ -1,5 +1,21 @@
 load("@rules_cc//cc:defs.bzl", "cc_library")
 
+copts_ = select({
```

Comment:
I like the idea of having the `copts_` and the select here. I'd like to ask you to do something like (in pseudo code):

```
copts_common_ = ["-Werror"]
copts_ = copts_common_ + select(...)
```

This would allow us to very clearly see what is common flags and what are platform specific flags.

If also paves the way for another PR where you can add compiler specific flags (again in pseudo code):

```
copts_ = copts_common_ + select(
 # os specific opts
) + select(
 # compiler specific opts
)
```

Reply:
Ok:) I like your idea about adding copts_common options. On my way to apply it!